Copyright (c) 2000-2007 Niels Provos <provos@citi.umich.edu>
Copyright (c) 2007-2012 Niels Provos and Nick Mathewson

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions
are met:
1. Redistributions of source code must retain the above copyright
   notice, this list of conditions and the following disclaimer.
2. Redistributions in binary form must reproduce the above copyright
   notice, this list of conditions and the following disclaimer in the
   documentation and/or other materials provided with the distribution.
3. The name of the author may not be used to endorse or promote products
   derived from this software without specific prior written permission.

THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR
IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT,
INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

Date: Nov 01 13:18:21 2017

From: 4ed348f478562b65578b3cccb3aff8f66c80d3ef Nov 02 14:21:35 2017

Subject: [PATCH] RDKB-14734 : Update LibEvent to Export Headers to Libevent Folder

	Libev and Libevent have folder conflict

Source: COMCAST

Upstream-Status: Pending

Signed-off-by: 	Shubham Baheti <shubham_baheti@comcast.com>
---

Index: libevent-2.0.21-stable/Makefile.am
===================================================================
--- libevent-2.0.21-stable.orig/Makefile.am
+++ libevent-2.0.21-stable/Makefile.am
@@ -226,7 +226,7 @@ noinst_HEADERS = util-internal.h mm-inte
 EVENT1_HDRS = event.h evhttp.h evdns.h evrpc.h evutil.h
 
 if INSTALL_LIBEVENT
-include_HEADERS = $(EVENT1_HDRS)
+pkginclude_HEADERS = $(EVENT1_HDRS)
 else
 noinst_HEADERS += $(EVENT1_HDRS)
 endif
